hw2/04-background.html

```diff
@@ -6,24 +6,31 @@
     <title>04 - Background</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="04-background.js"></script>
   </head>
-  <body>
-    <main>
+  <body id="background" class="row justify-content-center">
+    <main class="w-75 bg-light mt-5 p-4">
       <h1>Exercise 04 - Background</h1>
       <p>
         The background color will change every few seconds (as indicated below).
         You can stop this behavior and you can also change how often the
         background color changes.
       </p>
       <div>
-        <label>Background change interval (seconds): </label>
-        <input />
+        <label for="interval">Background change interval (seconds): </label>
+        <input type="number" id="interval" />
       </div>
       <div>
-        <input />
+        <input
+          type="button"
+          id="start-stop"
+          value="Start"
+          class="btn-primary"
```

Comment:
**Bootstrap Usage:**
The form elements and button use basic Bootstrap classes (btn-primary, bg-light). However, the button lacks the full set of Bootstrap button classes. You should use **btn btn-primary** instead of just **btn-primary** for consistent styling.
It would be better to utilize more Bootstrap classes for layout and spacing to ensure the form looks good **on all devices.**